### ISSUE TITLE
About the result！

### ISSUE THREAD

**Issue by hongge831** [xialeiliu/GFR-IL#1]
Hi, nice work first. I just run your code follow your steps in the README. But on cifar100 the result I got is worse than the result of your paper. here is my test result after training.
`Test starting -->	
Test 0	
Accuracy of Model 0 on Task 0 with unknown task boundary is 0.780
Average: 0.780      Weighted Average: 0.780
Test 1	
Accuracy of Model 1 on Task 0 with unknown task boundary is 0.714
Accuracy of Model 1 on Task 1 with unknown task boundary is 0.706
Average: 0.710      Weighted Average: 0.713
Test 2	
Accuracy of Model 2 on Task 0 with unknown task boundary is 0.699
Accuracy of Model 2 on Task 1 with unknown task boundary is 0.658
Accuracy of Model 2 on Task 2 with unknown task boundary is 0.630
Average: 0.662      Weighted Average: 0.690
Test 3	
Accuracy of Model 3 on Task 0 with unknown task boundary is 0.673
Accuracy of Model 3 on Task 1 with unknown task boundary is 0.588
Accuracy of Model 3 on Task 2 with unknown task boundary is 0.502
Accuracy of Model 3 on Task 3 with unknown task boundary is 0.644
Average: 0.602      Weighted Average: 0.651
Test 4	
Accuracy of Model 4 on Task 0 with unknown task boundary is 0.641
Accuracy of Model 4 on Task 1 with unknown task boundary is 0.556
Accuracy of Model 4 on Task 2 with unknown task boundary is 0.506
Accuracy of Model 4 on Task 3 with unknown task boundary is 0.576
Accuracy of Model 4 on Task 4 with unknown task boundary is 0.696
Average: 0.595      Weighted Average: 0.625
Test 5	
Accuracy of Model 5 on Task 0 with unknown task boundary is 0.626
Accuracy of Model 5 on Task 1 with unknown task boundary is 0.524
Accuracy of Model 5 on Task 2 with unknown task boundary is 0.480
Accuracy of Model 5 on Task 3 with unknown task boundary is 0.538
Accuracy of Model 5 on Task 4 with unknown task boundary is 0.586
Accuracy of Model 5 on Task 5 with unknown task boundary is 0.672
Average: 0.571      Weighted Average: 0.604
Test 6	
Accuracy of Model 6 on Task 0 with unknown task boundary is 0.605
Accuracy of Model 6 on Task 1 with unknown task boundary is 0.516
Accuracy of Model 6 on Task 2 with unknown task boundary is 0.444
Accuracy of Model 6 on Task 3 with unknown task boundary is 0.488
Accuracy of Model 6 on Task 4 with unknown task boundary is 0.638
Accuracy of Model 6 on Task 5 with unknown task boundary is 0.600
Accuracy of Model 6 on Task 6 with unknown task boundary is 0.552
Average: 0.549      Weighted Average: 0.580
Test 7	
Accuracy of Model 7 on Task 0 with unknown task boundary is 0.597
Accuracy of Model 7 on Task 1 with unknown task boundary is 0.514
Accuracy of Model 7 on Task 2 with unknown task boundary is 0.422
Accuracy of Model 7 on Task 3 with unknown task boundary is 0.466
Accuracy of Model 7 on Task 4 with unknown task boundary is 0.578
Accuracy of Model 7 on Task 5 with unknown task boundary is 0.572
Accuracy of Model 7 on Task 6 with unknown task boundary is 0.502
Accuracy of Model 7 on Task 7 with unknown task boundary is 0.712
Average: 0.545      Weighted Average: 0.573
Test 8	
Accuracy of Model 8 on Task 0 with unknown task boundary is 0.577
Accuracy of Model 8 on Task 1 with unknown task boundary is 0.498
Accuracy of Model 8 on Task 2 with unknown task boundary is 0.384
Accuracy of Model 8 on Task 3 with unknown task boundary is 0.468
Accuracy of Model 8 on Task 4 with unknown task boundary is 0.532
Accuracy of Model 8 on Task 5 with unknown task boundary is 0.550
Accuracy of Model 8 on Task 6 with unknown task boundary is 0.504
Accuracy of Model 8 on Task 7 with unknown task boundary is 0.636
Accuracy of Model 8 on Task 8 with unknown task boundary is 0.538
Average: 0.521      Weighted Average: 0.549
Test 9	
Accuracy of Model 9 on Task 0 with unknown task boundary is 0.562
Accuracy of Model 9 on Task 1 with unknown task boundary is 0.470
Accuracy of Model 9 on Task 2 with unknown task boundary is 0.410
Accuracy of Model 9 on Task 3 with unknown task boundary is 0.416
Accuracy of Model 9 on Task 4 with unknown task boundary is 0.490
Accuracy of Model 9 on Task 5 with unknown task boundary is 0.532
Accuracy of Model 9 on Task 6 with unknown task boundary is 0.472
Accuracy of Model 9 on Task 7 with unknown task boundary is 0.646
Accuracy of Model 9 on Task 8 with unknown task boundary is 0.504
Accuracy of Model 9 on Task 9 with unknown task boundary is 0.458
Average: 0.496      Weighted Average: 0.527
Test 10	
Accuracy of Model 10 on Task 0 with unknown task boundary is 0.554
Accuracy of Model 10 on Task 1 with unknown task boundary is 0.466
Accuracy of Model 10 on Task 2 with unknown task boundary is 0.370
Accuracy of Model 10 on Task 3 with unknown task boundary is 0.416
Accuracy of Model 10 on Task 4 with unknown task boundary is 0.510
Accuracy of Model 10 on Task 5 with unknown task boundary is 0.522
Accuracy of Model 10 on Task 6 with unknown task boundary is 0.462
Accuracy of Model 10 on Task 7 with unknown task boundary is 0.630
Accuracy of Model 10 on Task 8 with unknown task boundary is 0.464
Accuracy of Model 10 on Task 9 with unknown task boundary is 0.422
Accuracy of Model 10 on Task 10 with unknown task boundary is 0.550
Average: 0.488      Weighted Average: 0.518

Process finished with exit code 0
`
can your provide some other tricks for us? thanks in advance

**Comment by hongge831** [comment]
@xialeiliu, by the way, can you provide detailed environment requirements for us?

**Comment by xialeiliu** [comment]
Thanks for your feedback.

What you got is looking good actually. in the paper, we only reported the weighted average (51.8), which makes more sense than the average of all tasks (48.8), because we have 50 classes in the first task.

Environment we used:
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: linux-64
@EXPLICIT
https://conda.anaconda.org/pytorch/linux-64/cuda100-1.0-0.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/blas-1.0-mkl.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/ca-certificates-2019.1.23-0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/cudatoolkit-10.0.130-0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/intel-openmp-2019.3-199.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/libgcc-ng-8.2.0-hdf63c60_1.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/libgfortran-ng-7.3.0-hdf63c60_0.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/libstdcxx-ng-8.2.0-hdf63c60_1.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/libffi-3.2.1-hd88cf55_4.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/mkl-2018.0.3-1.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/ncurses-6.1-he6710b0_1.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/openssl-1.0.2r-h7b6447c_0.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/xz-5.2.4-h14c3975_4.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/zlib-1.2.11-h7b6447c_3.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/libedit-3.1.20181209-hc058e9b_0.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/readline-7.0-h7b6447c_5.tar.bz2
https://repo.continuum.io/pkgs/main/linux-64/tk-8.6.8-hbc83047_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/sqlite-3.28.0-h7b6447c_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/python-3.5.6-hc3d631a_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/certifi-2018.8.24-py35_1.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/ninja-1.8.2-py35h6bb024c_1.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/numpy-base-1.15.2-py35h81de0dd_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/pycparser-2.19-py35_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/cffi-1.11.5-py35he75722e_1.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/setuptools-40.2.0-py35_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/wheel-0.31.1-py35_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/pip-10.0.1-py35_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/mkl_fft-1.0.6-py35h7dd41cf_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/mkl_random-1.0.1-py35h4414c95_1.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/numpy-1.15.2-py35h1d66e8a_0.tar.bz2
https://conda.anaconda.org/pytorch/linux-64/pytorch-1.1.0-py3.5_cuda10.0.130_cudnn7.5.1_0.tar.bz2
